sda-commons-server-testing/src/test/java/org/sdase/commons/server/testing/junit5/EnvironmentExtensionTest.java

```diff
@@ -0,0 +1,40 @@
+package org.sdase.commons.server.testing.junit5;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+class EnvironmentExtensionTest {
```

Comment:
You're missing the pre- and post condition assertions of the original EnvironmentRuleTest. Don't we want to add them again?

Reply:
deleted